lib/libsecureboot/vets.c

```diff
@@ -568,9 +568,15 @@ verify_signer_xcs(br_x509_certificate *xcs,
 			ve_error_set("Validation failed, certificate not valid as of %s",
 			    gdate(date, sizeof(date), ve_utc));
 			break;
-		default:
-			ve_error_set("Validation failed, err = %d", err);
-			break;
+		default: {
```

Comment:
the formatting in this file looks off
did you run this through tools/build/checkstyle9.pl

Reply:
No I don't. I'll play with it ASAP.